.github/workflows/unit-test.yml

```diff
@@ -56,7 +56,13 @@ jobs:
           if [ "${{ matrix.node_version }}" = "18" ]; then
             export NODE_OPTIONS=--openssl-legacy-provider
           fi
-          npm run test
+
+          # only collect coverage when needed
+          if [ "${{ matrix.node_version }}" = "14" ]; then
```

Comment:
we should collect coverage on latest LTS (18 currently)

Reply:
I was wondering about that. Looked like 14 was picked on purpose (the step below used node 14 too). Anyway, I updated the workflow to 18 in [9507695](https://github.com/open-telemetry/opentelemetry-js/pull/3633/commits/950769508a2f7a49941e821223f29788a7693150) :slightly_smiling_face: